Create daily-crm-reporting-checklist.md (#89)
atolye15/checklist: checklists/daily-crm-reporting-checklist.md

---
title: Daily CRM Reporting Checklist
category: Project Management
date: "2020-03-12"
tags: ['crm', 'time-management']

> description: Follow the steps for the perfect CRM sharing! We request CRM from the team at the end of the everyday. Therefore, we can follow our performance (man/day) in each project.

Reviewer: ```suggestion
tags: ['worklog', 'time-management']
description: We request the worklog from the team at the end of every day so that, we can follow our performance (man/day) in each project, and we call it "CRM". Follow the steps for the perfect CRM sharing! 
```

---

- [ ] Write your project name which is time spent  
- [ ] Specify your time spent for each project  
You should specify it in 25 percent time frame. For example, you spent 45 minutes for your project. You should specify it as 0.75.  
- [ ] Specify what did you do in project  
You should specify your jira key ıd, title and status of task details.  
- [ ] Prefer Turkish characters while writing CRM  
- [ ] Check overall worktime of day  

> You should check overall worktime of day, it might be 7 hour and 30 minutes.

Reviewer: ```suggestion
- [ ] Write the project name as title that you're working on  
If you worked in multiple projects, we expect you to report them separately.

- [ ] Specify your work log for each project  
You should specify it in 15 minutes slots. 

- [ ] Specify what did you do in the project  
You should specify it with key-id and title that are taken from Jira. So that everyone in the Atolye15 might have an idea about what you're doing.

- [ ] Write down the status of your issues like "Done", "In Progress"

- [ ] It would be better to include Turkish characters while writing CRM  

- [ ] Check that the total time is equal to 7.30 hours.
We are counting the time over 7.30 hours.
```

---
title: Daily CRM Reporting Checklist
category: Project Management
date: "2020-03-12"
tags: ['worklog', 'time-management']

> description: We request the worklog from the team at the end of every day so that, we can follow our performance (man/day) in each project, and we call it "CRM". Follow the steps for the perfect CRM sharing! 

Reviewer 2: ```suggestion
description: We request the worklog from the team at the end of every day. So that, we can follow our performance (man/day) in each project and we call it "CRM". Follow the steps for the perfect CRM sharing! 
```

---

- [ ] Write the project name as title that you're working on  
If you worked in multiple projects, we expect you to report them separately.

> - [ ] Specify your work log for each project  

Reviewer 3: ```suggestion
- [ ] Specify your worklog for each project  
```

> You should specify it in 15 minutes slots. 

Reviewer 3: ```suggestion
You should round up or down the minutes to 15 minutes.
```

> - [ ] Specify what did you do in the project  

Reviewer 3: ```suggestion
- [ ] Specify the tasks/issues you have worked  
```

You should specify it with key-id and title that are taken from Jira. So that everyone in the Atolye15 might have an idea about what you're doing.

- [ ] Write down the status of your issues like "Done", "In Progress"

> - [ ] It would be better to include Turkish characters while writing CRM  

Reviewer 3: ```suggestion
```

> - [ ] Check that the total time is equal to 7.30 hours.

Reviewer 3: ```suggestion
- [ ] Check that the total time is equal to 7.30 hours  
```

title: Daily CRM Reporting Checklist
category: Project Management
date: "2020-03-12"
tags: ['worklog', 'time-management']

> description: We request the worklog from the team at the end of every day so that, we can follow our performance (man/day) in each project, and we call it "CRM". Follow the steps for the perfect CRM sharing! 

Reviewer 3: ```suggestion
description: We request the worklog from the team at the end of every day so that, we can follow our performance in each project. Follow the steps for the perfect CRM sharing! 
```

---

> - [ ] Write the project name as title that you're working on  

Reviewer 3: ```suggestion
- [ ] Write the project's name that you're working on as title  
```
